src/main/java/converters/IntConverter.java

```diff
@@ -0,0 +1,178 @@
+package converters;
+
+import exceptions.ConverterException;
+import options.IOption;
+import options.IntOption;
+
+import java.util.Objects;
+
+import static options.IntOption.BIG;
+import static options.IntOption.LITTLE;
+
+/**
+ * This class provides methods for converting between unsigned integer value strings and binary strings.
+ * It extends the abstract {@link Converter} class.
+ * <p>
+ * To convert an unsigned integer string to a binary string, use the {@link #convertTo(String, IOption[])} method.
+ * To convert a binary string to an unsigned integer value string, use the
+ * {@link #convertFrom(String, IOption[])} method.
+ * <p>
+ * The IntConverter supports the following options:
+ * <ul>
+ *     <li>{@link IntOption#BIG}: big-endian format (default)</li>
+ *     <li>{@link IntOption#LITTLE}: little-endian format</li>
+ * </ul>
+ */
+public class IntConverter extends Converter {
+
+	private static final int TEN = 10;
```

Comment:
thought: Is it really necessary to substitute this particular number? Does it really make the code more readable?